lib/plug/exceptions.ex

```diff
@@ -32,6 +36,29 @@ defmodule Plug.BadRequestError do
   defexception message: "could not process the request due to client error", plug_status: 400
 end
 
+defmodule BadRequestErrorHandler do
+  def print_hi do
+    IO.puts("HIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII")
+  end
+end
+
+defimpl Plug.Exception, for: Plug.BadRequestError do
+  def status(_), do: :bad_request
+
+  def actions(_exception) do
+    %{
+      print_hi_to_terminal: %{
```

Comment:
We we planning to use the key of the exception but we changed plans along the way. Now I am thinking if we should change how we represent actions. Maybe it should just return a list of maps?

```
def actions(_exception) do
  [
    %{
      label: "PRINT HIZINHO",
      action: ...
    }
  ]
end
```

Thoughts?